frontend/src/controllers/post_button_controller.js

```
@@ -3,9 +3,37 @@ import { showMessage } from "../utils/messages";
+  }
+
+  applySettings(event) {
+    // If an event is passed, check if it's for the correct project
+    if (event && event.detail.projectId !== this.projectIdValue) {
+      return;
+    }
+
+    const settingsJSON = localStorage.getItem(`userSettings:${this.projectIdValue}`);
+    if (settingsJSON) {
+      const settings = JSON.parse(settingsJSON);
+      if (settings.project && settings.project.has_auto_submission_setting) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)